test/runners/julia_spec.js

```diff
@@ -1,48 +1,43 @@
 var expect = require('chai').expect;
 var runner = require('../runner');
 
-describe('julia runner', function () {
-    // These specs are compatible with both Julia 0.4.6
-        describe('.run', function () {
-            it('should handle basic code evaluation', function (done) {
-                runner.run({
-                    language: 'julia',
-                    code: 'print("42")'
-                }, function (buffer) {
-                    console.log(buffer);
-                    expect(buffer.stdout).to.equal('42');
-                    done();
-                });
-            }); 
-       });
-        describe('Test', function () {
-            it('should handle a basic assertion', function (done) {
-                    runner.run({
-                        language: 'julia', 
-                        code: 'a = 1', 
-                        fixture:'include("frameworks/julia/Test.jl")\nusing Test\nfacts(()->(@fact a => 1))',
-                        testFramework: 'Test'
-                    }, function (buffer) {
-                        expect(buffer.stdout).to.equal('("<PASSED::>Test Passed","<:LF:>\\n")');
-                        done();
-                    });
-                });
+describe('julia runner', function() {
+  // These specs are compatible with both Julia 0.4.6
+  describe('.run', function() {
+      it('should handle basic code evaluation', function(done) {
+        runner.run({
+          language: 'julia',
+          code: 'print("42")'
+        }, function(buffer) {
+          expect(buffer.stdout).to.equal('42');
+          done();
+        });
+      });
 
-                it('should handle a basic description', function (done) {
-                    runner.run({
-                        language: 'julia', 
-                        code: 'global a = 1', 
-                        fixture: 'include("frameworks/julia/Test.jl")\nusing Test\nfacts(()->(@fact  a => 1) ,"test")', 
-                        testFramework: 'Test'
-                    }, function (buffer) {
-                        expect(buffer.stdout).to.contain('<DESCRIBE::>test<:LF:>\n("<PASSED::>Test Passed","<:LF:>\\n")');
-                        done();
-                    });
-                });
+    it('should handle a basic assertion', function(done) {
+      runner.run({
+        language: 'julia',
+        code: 'a = 1',
+        fixture: `
+          facts(()->(@fact a => 1))
+        `,
+        testFramework: 'Test'
+      }, function(buffer) {
+        expect(buffer.stdout).to.equal('("<PASSED::>Test Passed","<:LF:>\\n")');
```

Comment:
Apparently the format here is not correct per the comments. Since it comes from the Julia runner maybe you need to change that and adjust the test cases?